scalableminds/webknossos
# added version-protocol for foreign-datastore requests

### Steps to test:
- setup standalone-datastore
- comment out some routes from the datastore
- in BuildInfoSettings, change `"dataStoreApiVersion" to differ (in major version) for wK and the datastore to simulate different versions.
- provoke an error due to the commented out route
- A message should apper that the versions of webknossos and datastore do not match

### Issues:
- fixes #3087

------
